index.html

```diff
@@ -1179,8 +1179,14 @@ <h2>Securing Verifiable Credentials</h2>
         required to be secured or integrity protected or to contain a
         <code>proof</code> member.
       </p>
-      <p>Issuers, Holders, and Verifiers MUST ignore all JWT Claims Sets that
-        have no integrity protection.</p>
+      <p>
+        Issuers, Holders, and Verifiers MUST ignore all JWT Claims Sets that
+        have no integrity protection.
+      </p>
+      <p>
```

Comment:
@OR13 is this the correct place to add this guidance?

Reply:
This will work.